aviary/subsystems/propulsion/engine_deck.py

```diff
@@ -180,12 +184,20 @@ def __init__(self, name='engine_deck', options: AviaryValues = None,
         # Create dict for variables present in engine data with associated units
         self.engine_variables = {}
 
-        # TODO make this an option - disabling global throttle ranges is better to
-        #      prevent unintended extrapolation, but breaks missions using GASP-based
-        #      engines that have uneven throttle ranges (need t4 constraint on mission
-        #      to truly fix)
-        self.global_throttle = True
-        self.global_hybrid_throttle = True
+        if Aircraft.Engine.GLOBAL_THROTTLE in options:
```

Comment:
So it looks like these options are exclusive, they could both be true or both be false?

Reply:
Yeah you can have different values for hybrid vs. regular throttle